docs/data/data-grid/export/ExcelCustomExcelExport.tsx.preview

```diff
@@ -0,0 +1,11 @@
+<DataGridPro
```

Comment:
Can we rename the file to `CustomExelExport.tsx`?
X the other files.

Reply:
I propose `ExcelCustomExport`

I likfe prefixing by the "demo category", for the row grouping / tree data for instance, it helps visually splitting them into two groups.

Or we could create a sub-folder.